news/_posts/2021-06-15-multistore-news-in-1.7.8.md

```diff
@@ -0,0 +1,173 @@
+
+When checkboxes are not selected, the fields or toggles are disabled.
+
+Checkboxes allow:
+- to select one or many fields/toggles
+- to enable selected fields/toggles
+- to apply a different configuration for the selected shop or group
+- the merchant to see which parameters have a different value for the selected store or group
+
+![Multistore checkboxes](/assets/images/2021/06/multistore_checkboxes.png)
+ 
+For now, they have been implemented **only on the Maintenance page of the back office**. It will be implemented on the pages not yet migrated during the Symfony migration. For the pages already migrated in Symfony, we are looking for developers to implement it, so if you are willing to do this, please let us know :)
+ 
+## III.  Specific settings drop-down
+This drop-down is displayed in the “all shops” context and group contexts. It allows you to see which settings have been customized in one or several specific shops. And therefore, the settings that will not be overridden following a change made in “all shops” or in a group context.
+
+For now, it has been implemented **only on the Maintenance page of the back office**. It will be implemented on the pages not yet migrated during the Symfony migration. For the pages already migrated in Symfony, we are looking for developers to implement it, so if you are willing to do this, please let us know :)
+
+**Search bar** 
+
+
+In the specific settings drop-down, there is a search bar with auto-completion allowing you to search and select a shop or a group quickly. If the user chooses one of the listed results, it is automatically selected as context.
+
+In the specific settings drop-down there are:
+- The shop groups’ names with a link to select them as context
+- The shops’ names with a link to select them as context
+
+Next to each shop name, there is a mention:
+- "Inherited" if the parameter has the same value as in the “all shops” context
+- "Customized" if the parameter has a different value in a specific shop
+
+![Multistore specific settings dropdown](/assets/images/2021/06/multistore_specific_settings_dropdown.png)
+ 
+## IV.  Information messages
+Many information messages have been added on different pages and in different locations of the back office to provide better assistance, to inform of some behaviors as early as possible, and to avoid misunderstandings.
+
+For example, some pages are available only in the “all shops” context, and consequently, when you arrive on these pages, the context automatically switches to “all shops”. To prevent any issue, an information message has been added: "Note that this page is available in all shops context only, this is why your context has just switched."
+ 
+## V.  Modules
+Another pain identified during the user research step, was the difficulty of knowing whether a module is compatible with the multistore or not.
+
+To solve this problem the first thing to do was to define the notion of compatibility. A module can be :
```

Comment:
Maybe we could warn that in the BO, this information will be seen only in the module's description on the module manager page.  (which means that the module is already installed)

If one day this information becomes available from the marketplace too, we could add this information in the module's description from the catalog, but for now, it's not the case, unfortunately...